src/HotChocolate/Neo4J/test/HotChocolate.Data.Neo4J.Sorting.Tests/Strings/__snapshots__/Neo4JStringsSortingTests.Create_String_OrderBy.snap

```diff
@@ -14,12 +14,7 @@ ASC Result:
 }
 ---------------
 
-ASC Query:
----------------
-MATCH (fooString:FooString) RETURN fooString ORDER BY fooString.Bar ASC
----------------
-
-DESC Result:
+DESC
```

Comment:
Hmm shouldnt we keep the query in the snaps?

Reply:
I've found it helpful for the changes to paging because it confirms the neo4j query is written optimally against the test specimen. Definitely adds a layer of flakiness for any language changes but its a pretty stable language I think?

What do we currently do for other integrations?